src/System.Security.Cryptography.Algorithms/tests/ECDsaCngTests.cs

```diff
@@ -0,0 +1,224 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Security.Cryptography.EcDsa.Tests;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Cng.Tests
+{
+    public class ECDsaCngTests : ECDsaTestsBase
```

Comment:
This seems to have been copied, but not added to the project.  Probably just needs to be dropped out of the commit?